app/views/issues/_table.html.erb

```diff
@@ -58,18 +59,20 @@
                     sort: sort
                   }
               %>
-            <% end  %>
+            <% end %>
             <td class="column-actions">
               <%= link_to edit_project_issue_path(current_project, issue), class: '' do %>
                 <i class="fa fa-pencil"></i> Edit
               <% end %>
-              <%= link_to [current_project, issue], method: :delete, data: { confirm: "Are you sure?\n\nProceeding will delete this issue and any associated evidence." }, class: 'text-error-hover' do %>
-                <i class="fa fa-trash"></i> Delete
+              <% unless local_assigns[:hide_delete] %>
```

Comment:
have you tried this with caching? It seems like we're going to screw up the UI depending on whether a Delete row is chached is rendered first.